packages/common/src/utils/DateUtil.ts

```diff
@@ -26,9 +26,27 @@ const localeObject = {
   }
 }
 
+const MONTH_NAMES = [
+  'January',
```

Comment:
does indexing start from 0? or do you need an empty first element? 

ie, is Jan 1 or 0 ?

Reply:
Starts with 0, due to this;

```js
new Array(12)
        .fill(null)
        .map((_, idx) => idx)
        .reverse()
```